tests/ast/data/Session.create_dataframe_from_pandas.test

```diff
@@ -0,0 +1,18 @@
+## TEST CASE
+
+from snowflake.snowpark import Row
+
+from snowflake.snowpark.types import IntegerType, StringType, StructField
+
+import pandas as pd
+
+df = session.create_dataframe(pd.DataFrame([(1, 2, 3, 4)], columns=["a", "b", "c", "d"]))
+
+# Snowpark ignores provided schema for pandas Dataframe.
```

Comment:
Would it be possible to also test for the warning with these tests?

Reply:
we could certainly add a test! I imagine this to come up in code review for merge back, so let's dicuss with Snowpark python team then on what we should do for this case.